### DISCUSSION TITLE
Comms jamming: torque wrench SMA connectors

### DISCUSSION THREAD

**Discussion by newburgh** [dannyjacobs/ECHO#58] [General]
We think we jammed the drone communications signal when some of our (broadband) RF source leaked out from a non-torque-wrenched connection. We think the connection was only hand tightened and the connector jiggled a little loose during flight. In lab tests, the usual 5in-lb torque is sufficient for tightening during flight. 

(Hello world post from Laura)

**Comment by dannyjacobs** [comment]
Interesting! Your source goes all the way to 915MHz?  Or did you lose the
actual RC connection? Those are usually 2.5Ghz.

On Fri, Sep 16, 2022 at 5:31 PM newburgh ***@***.***> wrote:

> We think we jammed the drone communications signal when some of our
> (broadband) RF source leaked out from a non-torque-wrenched connection. We
> think the connection was only hand tightened and the connector jiggled a
> little loose during flight. In lab tests, the usual 5in-lb torque is
> sufficient for tightening during flight.
>
> (Hello world post from Laura)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dannyjacobs/ECHO/discussions/58>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPNV26BBSWM3OKSWATRI3V6SHFZANCNFSM6AAAAAAQON6BFY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
================================================================
Daniel C. Jacobs
 KE7DHQ
Assistant Professor
School of Earth and Space Exploration - Arizona State University
co-Director - Low frequency Cosmology Lab
Associate Director for Laboratories - Interplanetary Initiative
Project Scientist - HERA <http://reionization.org/>
Phone:           (505) 500 4521
Homepage:   danielcjacobs.com
